src/main/kotlin/com/ampnet/tradeservice/repository/OrderRepository.kt

```diff
@@ -0,0 +1,194 @@
+package com.ampnet.tradeservice.repository
+
+import com.ampnet.tradeservice.generated.jooq.enums.OrderStatus
+import com.ampnet.tradeservice.generated.jooq.enums.OrderType
+import com.ampnet.tradeservice.generated.jooq.tables.Order
+import com.ampnet.tradeservice.generated.jooq.tables.records.OrderRecord
+import com.ampnet.tradeservice.model.BlockchainOrderId
+import com.ampnet.tradeservice.model.InteractiveBrokersOrderId
+import com.ampnet.tradeservice.model.PlacedBuyOrder
+import com.ampnet.tradeservice.model.PlacedOrder
+import com.ampnet.tradeservice.model.PlacedSellOrder
+import com.ampnet.tradeservice.model.SerialId
+import org.jooq.DSLContext
+import org.jooq.TableField
+import org.springframework.stereotype.Repository
+import java.math.BigDecimal
+import java.time.Duration
+import java.time.OffsetDateTime
+
+@Repository
+@Suppress("TooManyFunctions")
+class OrderRepository(private val dslContext: DSLContext) {
+
+    fun storeBuyOrder(order: PlacedBuyOrder): SerialId<PlacedBuyOrder> {
```

Comment:
You could do this function without return like:
```suggestion
    fun storeBuyOrder(order: PlacedBuyOrder): SerialId<PlacedBuyOrder> = dsl...
```

Reply:
Yeah I know... I'm kind of used to using `=` syntax only for very short functions which have no side-effects and `{}` for everything else. But I will use `=` here.

Reply:
No problem... It was just a suggestion, you can leave it for the next time.